Fixes:

error: implicit declaration of function 'tgetent' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- cl/cl.h.orig	2023-09-22 20:32:31.000000000 -0700
+++ cl/cl.h	2023-09-24 00:34:31.000000000 -0700
@@ -20,6 +20,9 @@
 #else /* } { */
 #ifdef HAVE_NCURSES_H /* { */
 #include <ncurses.h>
+#include <term.h>
+#define HAVE_CURSES_SETUPTERM 1
+#define HAVE_CURSES_NEWTERM 1
 #else /* } { */
 #include <curses.h>
 #endif /* } */
